src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannelInsertionException.java

```diff
@@ -0,0 +1,19 @@
+package com.snowflake.kafka.connector.internal.streaming;
+
+/**
+ * Class for exceptions that occur while interacting with Snowflake through Streaming Snowpipe.
+ *
+ * <p>Use this exception when a particular channel (Topic Partition) fails to insert Rows into
+ * Snowflake Table.
```

Comment:
Client or Server side failure with respect to inserting rows?

Reply:
I dont differentiate it at the moment. So may be both.

Reply:
Ack, we may want to clarify that here.

Reply:
True that, may be I misunderstood where the exception was coming from. Most likely it was coming from Client SDK since the clientseq was bumped up and it never reached server. Let me clarify.